### PR TITLE
Migrate to Py3 & snapcrat.yaml complete

### PR DESCRIPTION
**Pull request by philroche**
* Strict confined snapcraft.yaml compelted successfully.
* Migrated code to python3
* Added new flag --config-skeleton to output example config format

### CODE REVIEW

File: snap/wrapper [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+# exec pip freeze
```
Comment (fginther) [inline snap/wrapper]: Remove this.

File: src/review_gator/review_gator.py [hunk 2]
```diff
@@ -417,24 +424,46 @@ def get_repos(sources):
     # XXX: Generate a warning if no user and password are found
```
Comment (fginther) [inline src/review_gator/review_gator.py]: This comment can be removed

File: templates/reviews.html [hunk 3]
```diff
@@ -1,129 +0,0 @@
-<!DOCTYPE html>
```
Comment (fginther) [inline templates/reviews.html]: Where did this file go?

File: snap/wrapper (hunk 1 above)
Comment (philroche) [reply snap/wrapper]: Done

File: src/review_gator/review_gator.py (hunk 2 above)
Comment (philroche) [reply src/review_gator/review_gator.py]: Done

**Comment by philroche** [comment]
Changes made and ready for review

**Pull request merged**